standalone/src/models/attribute-group/index.ts

```diff
@@ -5,3 +5,4 @@ export * as presets from './presets';
 export * from './types';
 
 export * from './attribute-group-draft';
+export * from './attribute-reference';
```

Comment:
We also export this model so consumers can use it in case they need to provide some custom values to the `AttributeGroupDraft`.